topics/icrc_25_signer_interaction_standard.md

```diff
@@ -151,19 +130,21 @@ While processing the request from the relying party, the signer can cancel it at
 
 1. The relying party sends a `icrc25_request_permissions` message to the signer.
 2. The signer removes any unrecognized scopes from the array of requested scopes.
-3. Depending on the session state the signer either skips or displays the details of the to-be-established connection to the user:
+3. Depending on whether the signer supports sessions:
+   - If the signer does _not_ support sessions, it may send a response immediately _without_ prompting the user. Scopes should be granted subject to a static signer policy. Skip to step 6.
+     > **Note:** Stateless signers MUST prompt the user to approve each request individually for each method defined in an ICRC-25 extension.
+   - Otherwise, continue with step 4.
+   > **Note:** It is recommended that signers assist users when granting permissions to relying parties, e.g. by maintaining a list of well-known relying parties and displaying additional information about the relying party, such as its name, logo, etc., or in the case of an unknown relying party, by displaying a warning.
```

Comment:
Why are we adding a recommendation for such a substantial centralized effort?

Reply:
I mean, I think it is a good recommendation (if the resources to implement it are available). The list must not necessarily be maintained in a centralized way...

But I see that this does not need to be in the standard necessarily. It is probably better suited in some additional document about signer implementation recommendations / best practices.

Removed.